tests/messages/mappers/test_bioscan_pool_xp_to_traction.py

```diff
@@ -0,0 +1,51 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from tol_lab_share.messages.consumed import BioscanPoolXpToTractionMessage
+from tol_lab_share.messages.mappers.bioscan_pool_xp_to_traction import BioscanPoolXpToTractionMapper
+from tests.data.example_bioscan_pool_xp_to_traction_messages import (
+    VALID_BIOSCAN_POOL_XP_TO_TRACTION_PAYLOAD as VALID_PAYLOAD,
+)
+
+
+@pytest.fixture
+def bioscan_pool_xp_message():
+    return BioscanPoolXpToTractionMessage(VALID_PAYLOAD)
+
+
+@pytest.fixture
+def mock_traction_message():
+    mock = MagicMock()
+    mock.create_request.return_value = MagicMock()
+    return mock
+
+
+@pytest.fixture
+def subject():
+    return BioscanPoolXpToTractionMapper()
+
+
+class TestBioscanPoolXpToTractionMapper:
+    def test_map_transfers_values(self, subject, bioscan_pool_xp_message, mock_traction_message):
```

Comment:
A general question. In cases like this is it possible to test for object parity? Do you prefer to test explicity for each field? Does that make it more difficult to maintain?

Reply:
The objects are not identical.  The values are transferred, but the layout of the objects are different.  If they weren't, we wouldn't need a mapper.